<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quản lý Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblID">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTen">
        <property name="text">
         <string>Tên:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtTen"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblMoTa">
        <property name="text">
         <string>Mô tả:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtMoTa"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblGia">
        <property name="text">
         <string>Giá:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtGia"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblLoai">
        <property name="text">
         <string>Loại:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtLoai"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblTrangThai">
        <property name="text">
         <string>Trạng thái:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtTrangThai"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableMenu">
      <property name="columnCount">
       <number>6</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Thêm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Sửa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Xóa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Làm mới</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStatistics">
        <property name="text">
         <string>Thống kê</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPay">
        <property name="text">
         <string>Thanh toán</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="btnConfirmPayment"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>